src/inc/clrconfigvalues.h

```diff
@@ -836,6 +836,7 @@ RETAIL_CONFIG_DWORD_INFO(UNSUPPORTED_ProfAPI_ValidateNGENInstrumentation, W("Pro
 
 #ifdef FEATURE_PERFMAP
 RETAIL_CONFIG_DWORD_INFO_EX(EXTERNAL_PerfMapEnabled, W("PerfMapEnabled"), 0, "This flag is used on Linux to enable writing /tmp/perf-$pid.map. It is disabled by default", CLRConfig::REGUTIL_default)
+RETAIL_CONFIG_DWORD_INFO_EX(EXTERNAL_PerfMapIgnoreSignal, W("PerfMapIgnoreSignal"), 0, "When perf map is enabled, this option will configure the specified signal to be accepted and ignored.  This allows it to be used as a marker in the perf logs.  It is disabled by default", CLRConfig::REGUTIL_default)
```

Comment:
I don't have a strong object to this, but I'm wondering if this should be associated with the perf map stuff (as opposed to not being under `FEATURE_PERFMAP` and just being called `COMPlus_IgnoreSignal` or something). I understand that your use case is profiling-associated, but perhaps we can make this a more general concept.